### DISCUSSION TITLE
SEB Support

### DISCUSSION THREAD

**Discussion by ano010** [SafeExamBrowser/seb-server#61] [General]
Hi SEB team,

First of all thanks for the product you are developing. We find it useful as it can be used to centralize and manage exams and users in one place especially the proctoring feature with Jitsi or Zoom significant. We have done the testing and initial POC. I am curious and would like to know more about the organization and people behind the SEB server development. May I know the capacity of the development team? How many people are working on SEB development? What challenges you are facing currently, about the next releases and features. Can we expect long-term support from the SEB development team like bug fixing, patching, feature releases, etc.?

Thanks
Anojan

**Comment by anhefti** [comment]
Hello Anojan

Thank you for your interest in SEB and SEB Server, we are very glad to hear that the POC was accepted and considered to be useful by you and your organization.
And please apologize the late answer. I was on holidays and have read your question just today. 

The SEB and SEB Server projects are funded and organized by the [SEB Alliance](https://safeexambrowser.org/alliance/members.html) 
The SEB Alliance is the body which sustains ongoing funding of the Safe Exam Browser open source project to continue its maintenance, development and support activities. ETH Zurich provides the infrastructure for the management and the software engineering of the SEB project and appoints an alliance manager who will provide administrative support to the SEB Alliance, and ensure the day-to-day running of the SEB Alliance. ETH Zurich leads the Alliance and offers different contribution levels to parties interested in the evolution of the SEB open source project. Direct consulting for SEB and SEB Server is exclusively for SEB Alliance contributors and depends on the contributor-level.  Currently there are three main developer working on the SEB and SEB Server project. More information about [joining](https://safeexambrowser.org/alliance/join.html) the Alliance is available in our [benefits](https://safeexambrowser.org/alliance/benefits.html) and [documents](https://safeexambrowser.org/alliance/documents.html) section.

I hope this answer the most of your questions. If you still have some questions, please don't hesitate to ask. Perhaps a discussion with the SEB Alliance would be interesting, too?

Best regards

**Reply by ano010** [reply]
Hi @anhefti,

Thanks for sharing the details. It would be really helpful I believe.

**Comment by anhefti** [comment]
@ano010 

Fine, and thanks again for your interest. 
If you don't mind, please send your request to the [SEB Alliance](https://safeexambrowser.org/alliance/join.html) and describe your points of interest and what you would like to discuss. We then can arrange a  Zoom Meeting for a personal discussion with the alliance manager and SEB developers.
